lib/jobs/run-graph.js

```diff
@@ -79,7 +80,8 @@ function runGraphJobFactory(
             self.graphTarget,
             self.options.graphName,
             self.options.graphOptions,
-            self.domain
+            self.domain,
+            self.proxy
```

Comment:
Would be great if you could add the same updates to https://github.com/RackHD/on-tasks/blob/master/lib/jobs/run-sku-graph.js#L97 as well!